cli/cmd/start.go

```diff
@@ -22,7 +22,7 @@ var (
 // NewStartCmd creates start command.
 func NewStartCmd() *cobra.Command {
 	var startCmd = &cobra.Command{
-		Use:   "start <APPLICATION_NAME>",
+		Use:   "start [<APPLICATION_NAME>]",
```

Comment:
I see a contradiction in start/stop/restart/status help messages... Why is start argument called APPLICATION_NAME? I can specify app or app:instance here. Perhaps, this is correct. But the same arg in status command is called INSTANCE_NAME. Why? I also can use app or app:instance as for the `start` command.
My suggestion is to use something like:
`[<APP_NAME> | <APP_NAME:INSTANCE_NAME>]`

Reply:
Agree, fixed.